README.md

```diff
@@ -82,15 +82,16 @@ If your data provider follow this convention, the importer can automatically cre
 The ```@en``` is a preset name, you must configuration the presets on your ```Settings.yaml```:
 
 ```
-Hsso:
-  Importer:
-    dimensionPresets:
-      fr:
-        language: ['fr', 'en', 'de']
-      en:
-        language: ['en', 'de']
-      de:
-        language: ['de']
+Ttree:
+  ContentRepositoryImporter:
+    dimensionsImporter:
+      presets:
+        fr:
+	  language: ['fr', 'en', 'de']
+        en:
+	  language: ['en', 'de']
```

Comment:
Identation issue, no ?

Reply:
Looks wrong in this view. But seems to be correct in the resulting document: https://github.com/dlubitz/ContentRepositoryImporter/blob/9fb7a13a4800573c1b55201480e4edc77e08b3eb/README.md